components/Sections/NewFooter/index.tsx

```diff
@@ -35,22 +35,22 @@ const Footer = () => (
           href="https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct"
           rel="noopener noreferrer"
         >
-          <FormattedMessage id="components.footer.links.codeOfConduct" />
+          <FormattedMessage id="components.section.newFooter.codeOfConduct" />
         </a>
       </li>
       <li>
         <LocalizedLink className={styles.link} href="/about/security">
-          <FormattedMessage id="components.footer.links.security" />
+          <FormattedMessage id="components.section.newFooter.security" />
         </LocalizedLink>
       </li>
       <li>
         <LocalizedLink className={styles.link} href="/about">
-          <FormattedMessage id="components.footer.links.about" />
+          <FormattedMessage id="components.section.newFooter.about" />
```

Comment:
You forgot to update the usage of `section` to `sections`. Again, be extra attentive when submitting changes...